inst/examples/ex01-basic/app.R

```diff
@@ -17,8 +45,10 @@ ui <- fluidPage(
 
 server <-  function(input, output, session) {
 
-  path <- system.file("examples", "ex01-basic", "parmesan", package = "parmesan")
-  parmesan <- parmesan_load(path)
+  # path <- system.file("examples", "ex01-basic", "parmesan", package = "parmesan")
+  # parmesan <- parmesan_load(path)
+  parmesan <- parmesan_load("parmesan")
```

Comment:
No need to remove how we load files in all examples. Use devtools::load_all of crtl + shift + L to test locally